dashboard/src/index.css

```
@@ -624,3 +624,155 @@
+  background: rgba(255, 255, 255, 0.12);
+  border-radius: 9999px;
+}
+.docs-scroll::-webkit-scrollbar-thumb:hover {
+  background: rgba(255, 255, 255, 0.22);
+}
+.docs-scroll::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+/* Motion: one orchestrated load reveal + the signature EKG pulse draw. */
+@keyframes docsRise {
```

Stylelint (17.12.0)

[error] Expected keyframe name "docsRise" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)